be/src/block_cache/block_cache.h

```diff
@@ -22,6 +24,26 @@
 
 namespace starrocks {
 
+class BlockCacheHitRateCounter {
```

Comment:
How about to move this class to another independent file? Because it actually save the upper layer statistics and has nothing to do with the underly block cache module. Also, it is unnecessary to include the bvar header to other modules which use block cache.

Reply:
changed